layers/+lang/c-c++/packages.el

```diff
@@ -223,7 +223,14 @@
     (spacemacs/add-realgud-debugger mode "gdb")))
 
 (defun c-c++/post-init-semantic ()
-  (spacemacs/add-to-hooks 'semantic-mode c-c++-mode-hooks))
+  ;; Disable semantic-idle-summary when use gtags layer.
+  ;; Gtags provides more useful information but can not do it properly
+  ;; when this mode is enabled since the minibuffer is cleared all the time."
+  (defun spacemacs//disable-semantic-idle-summary-mode ()
```

Comment:
I think this function can be extracted to `semantic` layer. Since it's being used in multiple layers.

Reply:
Done. Is it proper to put it into semantic's packages.el?